src/App.tsx

```diff
@@ -26,70 +27,130 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((g1, g2) => {
```

Comment:
```suggestion
  visibleGoods.sort((good1, good2) => {
```
I know, but better do not cut names)